github/commands/pull_request.py

```diff
@@ -30,8 +30,10 @@ def run(self):
         sublime.set_timeout_async(self.run_async, 0)
 
     def run_async(self):
-        base_remote = github.parse_remote(self.get_integrated_remote_url())
-        self.pull_requests = github.get_pull_requests(base_remote)
+        self.base_remote_name = self.get_integrated_remote_name()
```

Comment:
`self.get_integrated_remote_name()` is costly as it does at min 2 git calls, but the result is only used in a conditional once. Note that `get_integrated_remote_url` calls this internally.
```suggestion
```